### PR TITLE
Jungle Botany Redo 2 Electric Boogaloo

### PR DESCRIPTION
**Pull request by Addust**
## About The Pull Request

redesigns jungle botany to be Good
will write more later

## Why It's Good For The Game

old jungle botany was LITERALLY A KILO-CLASS SHIP _**WHAT THE FUCK**_

## Changelog

:cl:
add: new jungle botany! try not to FUCKING DIE!
/:cl:

### CODE REVIEW

**Comment by goober3** [comment]
image
thank u mapdiffbot, very helpful

**Review by spockye — COMMENTED** [review]
image

For being a botany ruin, the Botany element of it does feel kind of... minor to me. think you could make it a bit more central to the ruin, or atleast expand on it a bit more?

**Comment by PrefabQuasar** [comment]
please include a picture of the ruin in both your mapping software and ingame

File: _maps/map_catalogue.txt [hunk 1]
```diff
@@ -54,8 +54,8 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 
 	JungleRuins:
 		File Name = "_maps\RandomRuins\JungleRuins\jungle_botany.dmm"
-		Size = (x = 20)(y = 12)(z = 1)
-		Tags = "Minor Combat Challenge", "Medium Loot", "Liveable"
+		Size = (x = 45)(y = 60)(z = 1)
+		Tags = "Medium Combat Challenge", "High Loot", "Liveable"
```
Comment (Erikafox) [inline _maps/map_catalogue.txt:58]:
```suggestion
		Tags = "Medium Combat Challenge", "Major Loot", "Liveable"
```

File: code/game/area/areas/ruins/jungle.dm [hunk 2]
```diff
@@ -130,3 +130,37 @@
...
+
+/area/ruin/jungle/botanicalresearch/vault
+	name = "Forgotten Botanical Research Facility Vault"
+	icon_state = "storage"
+
+/area/ruin/jungle/botanicalresearch/checkpoint
+	name = "Forgotten Botanical Research Facility Security Checkpoint"
+	icon_state = "security"
+
+/area/ruin/jungle/botanicalresearch/caves
```
Comment (Erikafox) [inline code/game/area/areas/ruins/jungle.dm:160]: use /area/overmap_encounter/planetoid/cave/explored.

**Review by Erikafox — CHANGES_REQUESTED** [review]
https://github.com/shiptest-ss13/Shiptest/pull/1972#pullrequestreview-1461432521
I'm not gonna copy paste my entire review but this needs to be addressed

**Comment by spockye** [comment]
Creator does not intend on working on it

**Pull request closed without merging**